.github/workflows/main.yml

```diff
@@ -7,7 +7,45 @@ on:
     branches: [ main, opensuse ]
 
 jobs:
-  CI:
+  WithoutOptionalDeps:
+    runs-on: ubuntu-latest
+
+    container:
+      image: 'registry.fedoraproject.org/fedora:latest'
+      options: --security-opt seccomp=unconfined
+
+    steps:
+      - run: dnf --nogpgcheck --assumeyes install
+              /usr/bin/cpio
+              /usr/bin/bzip2
+              /usr/bin/python3
+              /usr/bin/readelf
+              /usr/bin/ldd
+              /usr/bin/c++filt
+              /usr/bin/xz
+              glibc
+              glibc.i686
+              python3-setuptools
+              python3-magic
+              python3-rpm
+              python3-pybeam
+              python3-pytest
+              python3-pytest-cov
+              python3-pytest-flake8
+              python3-pytest-xdist
+              python3-flake8
+              python3-flake8-import-order
+              python3-pyxdg
+              python3-toml
+              python3-zstd
+              python3-pip
+              rpm-build
+              git
+      - run: rm -rf $(rpm --eval %_dbpath)
```

Comment:
Please quote `%_dbpath` here so that the shell doesn't evaluate it.

Reply:
Fixed!